lib/Makefile

```diff
@@ -27,9 +27,16 @@ MODULE_big = pg_bulkload
 
 EXTENSION = pg_bulkload
 
+# set pg_bulkload version
+BULKLOAD_VERSION = 3.1.20
+
 #supports both EXTENSION (for >=9.1) and without_EXTENSION (for <PG 9.1)
-DATA_built = pg_bulkload.sql
-DATA = pg_bulkload--1.0.sql pg_bulkload--unpackaged--1.0.sql uninstall_pg_bulkload.sql 
+DATA_built = pg_bulkload.sql \
```

Comment:
Do we need pg_bulkload.sql and uninstall_pg_bulkload.sql?

Reply:
The file used to manually register the pg_bulkload schema and functions in postgresql. Someone might still be using it, so it is best not to delete it at this time.

Reply:
OK, understood. Thanks!